babel.config.js

```diff
@@ -17,7 +17,8 @@ module.exports = {
         realmModels: "./src/realmModels",
         sharedHelpers: "./src/sharedHelpers",
         sharedHooks: "./src/sharedHooks",
-        styles: "./src/styles"
+        styles: "./src/styles",
+        colors: "./tailwindColors"
```

Comment:
Can we just put this in styles and refer to it as `styles/tailwindColors`? I think that helps emphasize that it's one of our files and not some mysterious node module called `colors`.